aspnetcore/fundamentals/middleware/index.md

```diff
@@ -152,9 +164,7 @@ public void Configure(IApplicationBuilder app)
 }
 ```
 
-## Use, Run, and Map
-
-Configure the HTTP pipeline using <xref:Microsoft.AspNetCore.Builder.UseExtensions.Use*>, <xref:Microsoft.AspNetCore.Builder.RunExtensions.Run*>, and <xref:Microsoft.AspNetCore.Builder.MapExtensions.Map*>. The `Use` method can short-circuit the pipeline (that is, if it doesn't call a `next` request delegate). `Run` is a convention, and some middleware components may expose `Run[Middleware]` methods that run at the end of the pipeline.
```

Comment:
The first two lines here are already covered. I moved the last line to the section where I explain more about `Run`.